### PR TITLE
Journey preview

### PR DESCRIPTION
**Pull request by BrianLa0616**
### Summary 

This pull request implements the UI for previewing a journey.

- [x] implemented all designs

Remaining TODOs:

- [ ]  connect to back end

### CODE REVIEW

**Review by cathli66 — APPROVED** [review]
The preview looks great! Don't forget to resolve the merge conflict. You could consider changing the beat the timer toggle to blue when the user turns it on, as it's difficult to tell right now when toggling it off/on both are colored gray.

**Comment by youssef-attia** [comment]
Great design so far. I'm curious why we are adding the package_lock.json files here. Is that something that's needed for any of the widgets you're using or is it some other devops configuration?

File: game/lib/journeys/journey_cell.dart
Comment (erichuang27) [inline game/lib/journeys/journey_cell.dart:1]: This code is fire!

**Review by erichuang27 — DISMISSED** [review]
Awesome code, good work brian.

**Review by erichuang27 — APPROVED** [review]
Thank you for fixing the merge conflicts!

**Pull request merged**